src/bson.c

```diff
@@ -1201,6 +1289,195 @@ static bool php_phongo_bson_state_parse_type(zval* options, const char* name, ph
 	return retval;
 } /* }}} */
 
+static void field_path_map_element_set_info(php_phongo_field_path_map_element* element, php_phongo_bson_typemap_types type, zend_class_entry* ce TSRMLS_DC)
```

Comment:
`TSRMLS_DC` shouldn't be necessary here.

Reply:
Indeed - will remove it.